pyrometheus/__init__.py

```diff
@@ -216,8 +225,111 @@ def equilibrium_constants_expr(sol: ct.Solution, react: ct.Reaction, gibbs_rt):
 
 # {{{ Rate coefficients
 
-def arrhenius_expr(rate_coeff: ct.Arrhenius):
-    raise NotImplementedError()
+def rate_coefficient_expr(rate_coeff: ct.Arrhenius, t):
+    """:returns: The rate coefficient expression for *rate_coeff* in terms
+    of the temperature *t* as a :class:`pymbolic.primitives.Expression`"""
+    # Rate parameters
+    a = rate_coeff.pre_exponential_factor
+    b = rate_coeff.temperature_exponent
+    t_a = rate_coeff.activation_energy/ct.gas_constant
+    if b == 0 and t_a == 0:
+        # Constant rate
+        return a
+    elif b != 0 and t_a == 0:
+        # Weakly temperature-dependent rate
+        return a * t**b
+    elif b == 0 and a != 0 and t_a != 0:
+        # Classic Arrhenius rate
+        return p.Variable("exp")(np.log(a)-t_a/t)
+    else:
+        # Modified Arrhenius
+        return p.Variable("exp")(np.log(a)+b*p.Variable("log")(t)-t_a/t)
+
+
+def third_body_efficiencies_expr(sol: ct.Solution, react: ct.Reaction, c):
+    """:returns: The third-body concentration expression for reaction *react* in terms
+    of the species concentraions *c* as a :class:`pymbolic.primitives.Expression`"""
```

Comment:
This isn't quite right, syntax-wise. I've fixed this up in #12. (OK to resolve once #12 is merged into this.)

Reply:
I don't see a difference in #12 . I'm looking at this line:

https://github.com/ecisneros8/pyrometheus/blob/610bb688c08a8b55b4c47b88d107dd1374eb0610/pyrometheus/__init__.py#L232

Am I lost?

Reply:
I have fixed this up in #7

Reply:
Resolved?

Reply:
Yes, if @inducer had it fixed up in #12, then #7 resolves it.